distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Statement_Setter.enso

```diff
@@ -40,7 +43,16 @@ fill_hole_default stmt i value = case value of
         False -> stmt.setLong i value
     _ : Float   -> stmt.setDouble i value
     _ : Text      -> stmt.setString i value
-    _ : Date_Time -> JDBCUtils.setZonedDateTime stmt i value
+    _ : Date_Time ->
+        has_timezone = case type_hint of
+            Value_Type.Date_Time with_timezone -> with_timezone
+            # We include the timezone by default
+            _ -> True
+        case has_timezone of
+            True ->
+                JDBCUtils.setZonedDateTime stmt i value
+            False ->
+                JDBCUtils.setLocalDateTime stmt i value
```

Comment:
If the expected target type is `Date_Time with_timezone=False`, we use the special `setLocalDateTime` logic - to ensure that no timezone conversions are performed - but just the local date-time part is uploaded (the timezone is simply stripped from the value).

This is the best semantics we can do for values without timezone.